src/Libero/ruleset.xml

```diff
@@ -5,8 +5,25 @@
 
     <description>The Libero coding standard.</description>
 
-    <rule ref="PSR1"/>
+    <rule ref="PSR2"/>
 
+    <rule ref="SlevomatCodingStandard.TypeHints.ParameterTypeHintSpacing"/>
+    <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHintSpacing">
+        <properties>
+            <property name="spacesCountBeforeColon" value="1"/>
+        </properties>
+    </rule>
     <rule ref="Squiz.Classes.ClassFileName"/>
```

Comment:
No example for `trait`s yet, waiting for https://github.com/squizlabs/PHP_CodeSniffer/pull/2140 to the merged/released.